db/schema.rb

```diff
@@ -56,12 +56,12 @@
 
   create_table "custom_fields", force: :cascade do |t|
     t.datetime "created_at", null: false
-    t.integer "custom_field_set_id"
-    t.string "data_type"
-    t.string "internal_name"
-    t.string "label"
-    t.integer "parent_id"
-    t.integer "position"
+    t.integer  "custom_field_set_id"
```

Comment:
We're using `fix-db-schema-conflicts`, I have no idea why it doesn't seem to be doing its job here.